easybuild/easyconfigs/r/Reads2snp/Reads2snp-2.0.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'Tarball'
+
+name = 'Reads2snp'
+version = '2.0'
+
+homepage = 'http://kimura.univ-montp2.fr/PopPhyl/index.php?section=tools'
+description = """reads2snp is a SNP and genotype caller: it predicts the genotype of distinct individuals 
+ at distinct positions of a set of sequences based on read mapping / read counts. Its typical input is a bam file. 
+ Its typical output is a vcf file. It is written in C++, based on the bio++ libraries, multi-threaded with openMP, 
+ available under Linux and MacOS"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://kimura.univ-montp2.fr/PopPhyl/resources/tools/']
+sources = ['%(namelower)s_%(version)s.zip']
+
+sanity_check_paths = {
+    'files': ['reads2snp_2.0.64.bin'],
+    'dirs': [],
+}
+
+modextrapaths = {
+    'PATH': "", # add the installation dir to PATH
+}
+
+# disable exec permission for the provided 32 bits binary
+postinstallcmds = ['chmod -x %(installdir)s/reads2snp_%(version)s.32.bin']
```

Comment:
hmm, why? to prevent users from using the 32-bit version?

isn't this a site-specific customisation?